schema/phanbook.sql

```diff
@@ -15,13 +15,7 @@
 /*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 
---
--- Current Database: `phanbook`
---
 
-CREATE DATABASE /*!32312 IF NOT EXISTS*/ `phanbook` /*!40100 DEFAULT CHARACTER SET utf8 */;
-
-USE `phanbook`;
```

Comment:
When a user create a database name abc then import the file phanbook.sql it will be not import to database abc, instead of it will be created new database phanbook